utils/yaml-docker-execute.sh

```diff
@@ -102,7 +114,7 @@ do
       #
       # Run the checkov scan on the file
       #
-      checkov --download-external-modules True -f "${file}" >> ${REPORT_PATH} 2>&1
+      checkov ${CHECKOV_ARGS} --download-external-modules True -f "${file}" >> ${REPORT_PATH} 2>&1
```

Comment:
should `--download-external-modules True` be added to `CHECKOV_ARGS` above (following the same conventions?

Reply:
@climbertjh2 this one was weird; Checkov was throwing an unexpected type error for the `True` argument when passed in the same was as the others within the expanded variable, hence the special treatment here. Revisiting with fresh eyes.